alchemyjsonschema/__init__.py

```diff
@@ -2,6 +2,7 @@
 import logging
 from collections import OrderedDict
 import sqlalchemy.types as t
+import sqlalchemy.dialects.postgresql as pgt
```

Comment:
It's OK? use it with the environment that postgresql is not installed.

```console
$ python -m venv /tmp/foo
$ . /tmp/foo/bin/activate
$ pip install sqalchemy
$ python
Python 3.10.6 (main, Nov 14 2022, 16:10:14) [GCC 11.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import sqlalchemy.dialects
>>> import sqlalchemy.dialects.postgresql
```

OK

Reply:
Yes, this previously worked fine as long as the connected database was not postgresql, but given postgresql is a very commonly used database I think the addition of the JSON types used by postgresql was useful.